fiboa_cli/datasets/lu.py

```diff
@@ -0,0 +1,58 @@
+import os
+
+from ..convert_gml import gml_assure_columns
+from ..util import log
+import geopandas
+
+from ..convert_utils import convert as convert_
+
+SOURCES = {
+    "https://data.public.lu/fr/datasets/r/b4ae6690-7e4c-4454-8b60-9fa33ba6a61b": "lu.zip"
+}
+# SOURCES = "https://download.data.public.lu/resources/referentiel-des-parcelles-flik/20240126-145117/20240111-parcellesreference.zip"
+
+ID = "lu"
+SHORT_NAME = "Luxembourg"
+TITLE = "Luxembourg FLIK Parcels"
+DESCRIPTION = """
+The Land Parcel Identification System (LPIS) is a reference database of the agriculture parcels used as a basis for area related payments to farmers in relation to the Common Agricultural Policy (CAP). These payments are (co)financed by the European Agricultural Guarantee Fund (‘EAGF’) and the European Agricultural Fund for Rural Development (‘EAFRD’).
+To ensure that payments are regular, the CAP relies on the Integrated Administration and Control System (IACS), a set of comprehensive administrative and on the spot checks on subsidy applications, which is managed by the Member States. The Land Parcel Identification System (LPIS) is a key component of the IACS. It is an IT system based on ortho imagery (aerial or satellite photographs) which records all agricultural parcels in the Member States. It serves two main purposes: to clearly locate all eligible agricultural land contained within reference parcels and to calculate their maximum eligible area (MEA). The LPIS is used for cross checking during the administrative control procedures and as a basis for on the spot checks by the paying agency.
+"""
+
+
+PROVIDERS = [
+    {
+        "name": "Luxembourg ministry of Agriculture",
+        "url": "https://agriculture.public.lu/",
+        "roles": ["licensor", "distributor"]
+    }
+]
+ATTRIBUTION = "Luxembourg ministry of Agriculture"
+LICENSE = {
+    "title": "This layer is published under the terms of the license Creative Commons Attribution 4.0 International (CC BY 4.0)",
+    "href": "https://creativecommons.org/licenses/by/4.0/",
+    "type": "text/html",
+    "rel": "license"
+}
```

Comment:
This can be simplified.

```suggestion
LICENSE = "CC-BY-4.0"
```